library/src/test/java/com/okta/oidc/clients/sessions/SessionClientImplTest.java

```diff
@@ -232,7 +232,7 @@ public void refreshTokenFailure() throws InterruptedException {
         latch.await();
         assertNull(cb.getResult());
         assertNotNull(cb.getException());
-        assertEquals(cb.getException().getMessage(), "Invalid status code 401 Client Error");
+        assertEquals(cb.getException().getMessage(), "No client credentials found.");
```

Comment:
The original issue was saying we had multiple exceptions with the same type/code. Please add an assertion to confirm we fixed the original issue.

Reply:
Added the assertion below.